lte/gateway/c/core/oai/include/amf_service_handler.h

```diff
@@ -19,7 +19,6 @@
  */
 int send_n11_create_pdu_session_resp_itti(
     itti_n11_create_pdu_session_response_t* itti_msg);
-
```

Comment:
Please keep the new line here. No point committing a whitespace change as the only change in a file.

Reply:
addressed review comment